recipes/platform.interfaces/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.interfaces"
+    license = "The Unlicense"
+    homepage = "https://github.com/linksplatform/Interfaces"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Contains common interfaces that did not fit in any major category."

```

Comment:
Can you make the description more _self-contained_? This will be shown to users in ConanCenter webpage without any context, it could be useful to add something about LinksPlatform as well, maybe the "About" line:

```suggestion
    description = "LinksPlatform's Platform.Interfaces Class Library. Contains common interfaces that did not fit in any major category."

```

Reply:
Can I use the description from issue?
> Platform.Interfaces is one of the libraries of the LinksPlatform modular framework, which uses innovations from the C++20 standard, for easier use of static polymorphism.
It also includes some auxiliary structures for more convenient work with containers.

Reply:
Sure, why not? I was only suggesting that some more verbose would be more useful. 👍

Reply:
@jgsogo looks like this should be resolved, already :) Can you take a look?